.travis.yml

```diff
@@ -22,8 +22,6 @@ cache:
 
 matrix:
   include:
-    - python: 3.5
```

Comment:
do we have to drop 3.5 testing?  Although I initially suggested we do that, I thought we settled on removing the one f-string in a backport.

Reply:
It's not about language features we're using; it's about packages available from conda. (Note that this is a `CONDA_ENV=1` build.) However, now that we have (rightly!) removed the pyepics pin that was once in this PR, perhaps we can restore the 3.5 build and remove the `python_requires`.

Reply:
To underline the point: thanks for the close read, @tacaswell!

Reply:
fair cop.